.rubocop.yml

```diff
@@ -10,6 +10,14 @@ inherit_mode:
   merge:
     - Exclude
 
+Style/BlockDelimiters:
+  Exclude:
+    - app/components/induction_summary_component.rb
+
+Style/MethodCalledOnDoEndBlock:
+  Exclude:
+    - app/components/induction_summary_component.rb
```

Comment:
If these aren't temporary, might be worth disabling the rules locally in the file itself.

Reply:
Actually nvm, can see the benefit of keeping stuff like this in a single place. Ignore! 😄